pan3d/dataset_builder.py

```diff
@@ -27,7 +29,24 @@
 
 @TrameApp()
 class DatasetBuilder:
-    def __init__(self, server=None, dataset_path=None, state=None, pangeo=False):
+    """Manage data slicing, mesh creation, and rendering for a target N-D dataset."""
+
+    def __init__(
+        self,
+        server: trame_server.core.Server = None,
```

Comment:
This should either be a server or a name (str).

Reply:
Changed in [aa5dbc8](https://github.com/Kitware/pan3d/pull/47/commits/aa5dbc87edb31c90529218b433aa876d09299801)